rl4co/envs/scheduling/jssp.py

```diff
@@ -0,0 +1,665 @@
+from typing import Optional, Tuple
+
+import torch
+
+from tensordict.tensordict import TensorDict
+from torchrl.data import CompositeSpec, DiscreteTensorSpec, UnboundedContinuousTensorSpec
+
+from rl4co.envs.common.base import RL4COEnvBase
+
+
+class Configs:
+    low = 1
+    high = 99
+    et_normalize_coef = 1000
+    rewardscale = 0.0
+    init_quality_flag = False
+
+
+configs = Configs()
```

Comment:
About config: is this supposed not to be ever changed by the user?

Because if it can be changed, isn't it better to set these as hyperparameters to the `JSSPEnv` class?